Assets/Scripts/Sword.cs

```diff
@@ -14,13 +14,7 @@ void Start()
     {
         rigidbody = GetComponent<Rigidbody2D>();
         rigidbody.velocity = transform.right * speed;
-        StartCoroutine(DestroySword());
-    }
-
-    private IEnumerator DestroySword()
-    {
-        yield return new WaitForSeconds(5);
-        Destroy(gameObject);
+        Destroy(gameObject, 5f);
```

Comment:
I like this, had the same problem with mine before figuring this out

Reply:
5 is a magic number here